assets/src/styles/partials/_header.scss

```
@@ -26,5 +26,34 @@ header {
                 }
             }
         }
+        .banner-title {
+            font-family: calibri,tahoma,arial,verdana,sans-serif;
+            font-size: 1em;
+            text-decoration: none;
+            text-align: right;
+            display: inline-block;
+        
+            a {
+                color: #233f8f !important;
+            }
+            .ngwmn-highlight {
+                color: #d1544a;
+            }
+        }
+        @include at-media('mobile-lg') {
+            .banner-title {
+                font-size: 1.25em;
+            }
+        }    
+        @include at-media('tablet') {
+            .banner-title {
+                font-size: 1.5em;
+            }
+        }
+        @include at-media('desktop-lg') {
```

So I just looked into the default theme, and it turns out desktop-lg isn't available to us currently, so I don't think this has any effect. This should make it more similar to the original with the `px` usage.

```suggestion
         @include at-media('desktop') {
```

---

Should we adjust the reference above? It appeared to work in my local env.